src/services

```diff
@@ -1085,3 +1085,10 @@ p({
     }
   """
 })
+
+p({
```

Comment:
This is not required if we create one per DAO. 
Both GUIDAO and SeqNoDAO can be updated to use this generator. 
GUIDDAO saving as a String, and SeqNoDAO saving the Long.

Reply:
Transaction id is not set in GUIDAO or SeqNoDAO. It is set at many different places on nanopay side ( mainly in AbstractTransactionPlanner). Is this required to allow us to set the transaction id specifically? Or do we want to remove all those ids that are set specifically and let GUIDAO set the transaction id instead? @jlhughes @KernelDeimos